flamedisx/xenon/x1t_sr1.py

```diff
@@ -216,14 +230,14 @@ def add_extra_columns(self, d):
             d['cs2'] = (
                 d['s2']
                 / d['s2_relative_ly']
-                * np.exp(d['drift_time'] / self.defaults['elife']))
+                * np.exp(d['drift_time'] / d['elife']))
+
 
     @staticmethod
     def electron_detection_eff(drift_time,
+                               elife,
```

Comment:
As before, we should have the option for constant elife as well

Reply:
I have reintroduced the option of having constant elife using the variable `variable_elife` which will set elife to a constant in `add_extra_columns` (see https://github.com/FlamTeam/flamedisx/blob/d620b0fddba6612f2b67476c086b046ecf9d5b3b/flamedisx/xenon/x1t_sr1.py#L229).

However, it is not possible to let elife float as a nuisance parameter now. But according to [the convo note](http://links.app.convo.com/v1/acc-d87cde0a-409c-11e3-adfe-12313d0421bc/apps/6/messages/Y7CezL6ZgDwU7xOzATirnC76NGL1k8#title=Elife%252520bug%25253A%252520now%252520s2%252520fits%252520well%252520but%252520not%252520cs2%25253F) (which Jelle should be able to see well), keeping it fixed at 641us (analysis paper I value) gives a better fit than letting it float as a nuisance parameter.